src/main/webapp/resources/js/apis/projects/projects.js

```diff
@@ -31,3 +31,9 @@ export async function getPagedProjectsForUser(params) {
 export async function getProjectRoles() {
   return await axios.get(`${URL}/roles`).then(({ data }) => data);
 }
+
+// export async function createProject(parameters) {
```

Comment:
Remove commented out code

Reply:
Updated in 580b1d4f2538f68269716be54bea59fdf8ed3610